grasshopper-loader.js

```diff
@@ -119,34 +138,39 @@ function run(program, loaderCallback){
       source = ogrChild(fileName, stream, program.sourceSrs).stdout;
     }
 
-    var verifyResults = verify(fileName, stream);
+    if(path.extname(fileName) === '.csv') splitter = csvParse({columns: true});
+    else splitter = splitOGRJSON();
 
-    source
-      .pipe(splitOGRJSON())
-      .pipe(transformer(fileName, makeBulkSeparator(), '\n'))
-      .pipe(lump(Math.pow(2,20)))
-      .pipe(loader)
+    var verifyResults = verify(fileName, stream, scratchSpace, loaderCallback);
+
+    pump(
+      source,
+      splitter,
+      transformer(fileName, makeBulkSeparator(), '\n'),
+      lump(Math.pow(2, 20)),
+      loader,
+
+      function(err){
+        if(err){
+          console.log('Streaming of %s interrupted by error,', fileName);
```

Comment:
Seems like it would be useful to have to underlying error part of this message...or is this logged upstream somewhere by returning `loaderCallback(err)`?